<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1500</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>COCOMO</string>
  </property>
  <widget class="QComboBox" name="comboBox_1">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>20</y>
     <width>140</width>
     <height>22</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>2</number>
   </property>
   <item>
    <property name="text">
     <string>Очень низкий</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Низкий</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Номинальный</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Высокий</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Очень высокий</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="comboBox_2">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>50</y>
     <width>140</width>
     <height>22</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <item>
    <property name="text">
     <string>Низкий</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Номинальный</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Высокий</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Очень высокий</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>340</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>RELY (Требуемая надежность)</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>340</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>DATA (Размер базы данных)</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>340</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>CPLX (Сложность продукта)</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>340</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>TIME (Ограничение времени выполнения)</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>340</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>STOR (Ограничение объема основной памяти)</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>170</y>
     <width>340</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>VIRT (Изменчивость виртуальной машины)</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>200</y>
     <width>340</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>TURN (Время реакции компьютера)</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>230</y>
     <width>340</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>ACAP (Способности аналитика)</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>260</y>
     <width>340</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>AEXP (Знание приложений)</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>290</y>
     <width>340</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>PCAP (Способности программиста)</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>320</y>
     <width>340</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>VEXP (Знание виртуальной машины)</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>350</y>
     <width>340</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>LEXP (Знание языка программирования)</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>380</y>
     <width>340</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>MODP (Использование современных методов)</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>410</y>
     <width>340</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>TOOL (Использование программных инструментов)</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_15">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>440</y>
     <width>340</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>SCED (Требуемые сроки разработки)</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_3">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>80</y>
     <width>140</width>
     <height>22</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>2</number>
   </property>
   <item>
    <property name="text">
     <string>Очень низкий</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Низкий</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Номинальный</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Высокий</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Очень высокий</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="comboBox_4">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>110</y>
     <width>140</width>
     <height>22</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <item>
    <property name="text">
     <string>Номинальный</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Высокий</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Очень высокий</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="comboBox_5">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>140</y>
     <width>140</width>
     <height>22</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <item>
    <property name="text">
     <string>Номинальный</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Высокий</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Очень высокий</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="comboBox_6">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>170</y>
     <width>140</width>
     <height>22</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <item>
    <property name="text">
     <string>Низкий</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Номинальный</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Высокий</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Очень высокий</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="comboBox_7">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>200</y>
     <width>140</width>
     <height>22</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <item>
    <property name="text">
     <string>Низкий</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Номинальный</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Высокий</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Очень высокий</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="comboBox_8">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>230</y>
     <width>140</width>
     <height>22</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>2</number>
   </property>
   <item>
    <property name="text">
     <string>Очень низкий</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Низкий</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Номинальный</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Высокий</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Очень высокий</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="comboBox_9">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>260</y>
     <width>140</width>
     <height>22</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>2</number>
   </property>
   <item>
    <property name="text">
     <string>Очень низкий</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Низкий</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Номинальный</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Высокий</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Очень высокий</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="comboBox_10">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>290</y>
     <width>140</width>
     <height>22</height>
    </rect>
   </property>
   <property name="currentText">
    <string>Высокий</string>
   </property>
   <property name="currentIndex">
    <number>3</number>
   </property>
   <item>
    <property name="text">
     <string>Очень низкий</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Низкий</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Номинальный</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Высокий</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Очень высокий</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="comboBox_11">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>320</y>
     <width>140</width>
     <height>22</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>2</number>
   </property>
   <item>
    <property name="text">
     <string>Очень низкий</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Низкий</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Номинальный</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Высокий</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="comboBox_12">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>350</y>
     <width>140</width>
     <height>22</height>
    </rect>
   </property>
   <property name="currentText">
    <string>Высокий</string>
   </property>
   <property name="currentIndex">
    <number>3</number>
   </property>
   <item>
    <property name="text">
     <string>Очень низкий</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Низкий</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Номинальный</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Высокий</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="comboBox_13">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>380</y>
     <width>140</width>
     <height>22</height>
    </rect>
   </property>
   <property name="currentText">
    <string>Очень высокий</string>
   </property>
   <property name="currentIndex">
    <number>4</number>
   </property>
   <item>
    <property name="text">
     <string>Очень низкий</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Низкий</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Номинальный</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Высокий</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Очень высокий</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="comboBox_14">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>410</y>
     <width>140</width>
     <height>22</height>
    </rect>
   </property>
   <property name="currentText">
    <string>Высокий</string>
   </property>
   <property name="currentIndex">
    <number>3</number>
   </property>
   <item>
    <property name="text">
     <string>Очень низкий</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Низкий</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Номинальный</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Высокий</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Очень высокий</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="comboBox_15">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>440</y>
     <width>140</width>
     <height>22</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>2</number>
   </property>
   <item>
    <property name="text">
     <string>Очень низкий</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Низкий</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Номинальный</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Высокий</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Очень высокий</string>
    </property>
   </item>
  </widget>
  <widget class="QLineEdit" name="sizeEdit">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>500</y>
     <width>140</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>25</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_16">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>500</y>
     <width>340</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>KLOC (Размер программного кода в тысячах строк)</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_16">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>530</y>
     <width>140</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Обычный</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Промежуточный</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Встроенный</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_17">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>530</y>
     <width>340</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Режим проекта</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="wbsTable">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>40</y>
     <width>950</width>
     <height>365</height>
    </rect>
   </property>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>150</number>
   </attribute>
   <row>
    <property name="text">
     <string>Анализ требований</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>Проектирование продукта</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>Программирование</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>Планирование тестирования</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>Верификация и аттестация</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>Канцелярия проекта</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>Управление конфигурацией и обеспечение качества</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>Создание руководств</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>ИТОГО</string>
    </property>
   </row>
   <column>
    <property name="text">
     <string>Бюджет (%)</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Человеко-месяцы</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Затраты (тыс руб)</string>
    </property>
   </column>
   <item row="0" column="0">
    <property name="text">
     <string>4</string>
    </property>
   </item>
   <item row="1" column="0">
    <property name="text">
     <string>12</string>
    </property>
   </item>
   <item row="2" column="0">
    <property name="text">
     <string>44</string>
    </property>
   </item>
   <item row="3" column="0">
    <property name="text">
     <string>6</string>
    </property>
   </item>
   <item row="4" column="0">
    <property name="text">
     <string>14</string>
    </property>
   </item>
   <item row="5" column="0">
    <property name="text">
     <string>7</string>
    </property>
   </item>
   <item row="6" column="0">
    <property name="text">
     <string>7</string>
    </property>
   </item>
   <item row="7" column="0">
    <property name="text">
     <string>6</string>
    </property>
   </item>
   <item row="8" column="0">
    <property name="text">
     <string>100</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_18">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>10</y>
     <width>531</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Декомпозиция работ по созданию ПО</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="classicTable">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>470</y>
     <width>950</width>
     <height>291</height>
    </rect>
   </property>
   <attribute name="verticalHeaderVisible">
    <bool>true</bool>
   </attribute>
   <attribute name="verticalHeaderCascadingSectionResizes">
    <bool>false</bool>
   </attribute>
   <attribute name="verticalHeaderHighlightSections">
    <bool>true</bool>
   </attribute>
   <attribute name="verticalHeaderShowSortIndicator" stdset="0">
    <bool>false</bool>
   </attribute>
   <attribute name="verticalHeaderStretchLastSection">
    <bool>false</bool>
   </attribute>
   <row>
    <property name="text">
     <string>Планирование и определение требований</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>Проектирование продукта</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>Детальное проектирование</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>Кодирование и тестирование отдельных модулей</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>Интеграция и тестирование</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>Итого без планирования</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>ИТОГО</string>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
   </row>
   <column>
    <property name="text">
     <string>Трудозатраты (%)</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Трудозатраты</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Время (%)</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Время</string>
    </property>
   </column>
   <item row="0" column="0">
    <property name="text">
     <string>8</string>
    </property>
   </item>
   <item row="0" column="2">
    <property name="text">
     <string>36</string>
    </property>
   </item>
   <item row="1" column="0">
    <property name="text">
     <string>18</string>
    </property>
   </item>
   <item row="1" column="2">
    <property name="text">
     <string>36</string>
    </property>
   </item>
   <item row="2" column="0">
    <property name="text">
     <string>25</string>
    </property>
   </item>
   <item row="2" column="2">
    <property name="text">
     <string>18</string>
    </property>
   </item>
   <item row="3" column="0">
    <property name="text">
     <string>26</string>
    </property>
   </item>
   <item row="3" column="2">
    <property name="text">
     <string>18</string>
    </property>
   </item>
   <item row="4" column="0">
    <property name="text">
     <string>31</string>
    </property>
   </item>
   <item row="4" column="2">
    <property name="text">
     <string>28</string>
    </property>
   </item>
   <item row="5" column="0">
    <property name="text">
     <string>100</string>
    </property>
   </item>
   <item row="5" column="2">
    <property name="text">
     <string>100</string>
    </property>
   </item>
   <item row="6" column="0">
    <property name="text">
     <string>108</string>
    </property>
   </item>
   <item row="6" column="1">
    <property name="text">
     <string/>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
   </item>
   <item row="6" column="2">
    <property name="text">
     <string>136</string>
    </property>
   </item>
   <item row="6" column="3">
    <property name="text">
     <string/>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_19">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>440</y>
     <width>591</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Распределение работ и времени по стадиям жизненного цикла</string>
   </property>
  </widget>
  <widget class="QPushButton" name="projectButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>560</y>
     <width>240</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Рассчитать проект</string>
   </property>
  </widget>
  <widget class="QPushButton" name="task1Button">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>560</y>
     <width>240</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Решить задачу 1</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
